src/main/resources/css/common.css

```diff
@@ -141,7 +151,6 @@ span {
 }
 .unavailable {
     background: white !important;
-    border: none !important;
```

Comment:
why did we remove this style? on tablet and desktop now the cards have outlines even when unavailable, which goes against the designs i have from thomas?

Reply:
Oh my fault, I wasn't testing out for those cards and I thought it applied to chapters only.

Reply:
I added it back!